modules/installation/pages/install-proxy.adoc

```diff
@@ -1,8 +1,130 @@
+|                      | _Production Server_ Minimum 8{nbsp}GB
+| CPUs:                | 2
+| Storage Format:      | ISO 3{nbsp}GB
+| Disk Space:          | 230{nbsp}GB split between
+|                      | [path]``/`` _(root)_ Minimum 24{nbsp}GB
+|                      | (Virtual Disk 1) [path]``/srv`` Minimum 100{nbsp}GB
+|                      | (Virtual Disk 2) [path]``/var/cache`` _(Squid)_ Minimum 100{nbsp}GB
+| Name:                | example-proxy
+| Network              | Bridge br0
+|===
+
+
+
+[[installation-proxy-sles-settings]]
+=== SLES KVM Settings
+
+This section provides guidance on installation of {productname} Proxy
+utilizing the full installation media with KVM and [command]``virt-manager``.
+This section assumes you have previously setup an account with SCC and downloaded the SLES full installation media.
+
+
+
+.Procedure: Preparing for SLES Installation
+. In [command]``virt-manager`` select menu:File[New Virtual Machine].
+
+. Select btn:[Local install media (ISO image or CDROM)].
+
+. Ensure btn:[Use ISO Image] is selected.
+Then click btn:[Browse] and locate the full SLES image you downloaded from your SCC account.
+
+. Configure your machine with at least 2 GB RAM and a minimum of 2 CPUs.
+
+. Create a storage device with a minimum of 230 GB storage space for the installation.
+During the partitioning setup of the SLES installation this disk should be partitioned into the following disks:
++
+// FIXME: not sure about swap space
+// |4{nbsp}GB Swap space
++
+[cols="1", options="header"]
+|===
+| Disk Space Requirements
+| 100{nbsp}GB XFS partition (or dedicated virtual disk) for [path]``/srv/``
+| 100{nbsp}GB XFS partition (or dedicated virtual disk) for [path]``/var/cache/``
+|===
+
+. The remaining storage space will be used by the operating system for the root partition.
+Select btn:[Finish] to begin the installation.
+
+Installation of {sles} will begin.
+For more information on completing an installation of {sles}, see: https://documentation.suse.com/sles/15-SP1/html/SLES-all/art-sle-installquick.html.
+
+
+
+[[installation-proxy-prep]]
+== Prepare SLES for SUSE Manager Proxy
+
+[[proc.jeos.susemgr.prep]]
+.Procedure: Preparing SLES for {productname} Proxy Installation
+
+// Most steps are currently needed because of 4.0 workarounds
+. Log in as `root`.
+
+. Uninstall the `sles-release` package:
++
+
+----
+rpm -e --nodeps sles-release
+----
+
+. Register {productname} Proxy with SCC (for example, replace `<productnumber>` with `{productnumber}` and `<architecture>` with `x86_64`):
++
+
+----
+SUSEConnect -e<EMAIL_ADDRESS> -r<SUSE_MANAGER_PROXY_CODE> \
+  -p sle-module-suse-manager-proxy/<productnumber>/<architecture>
+----
+
+. Add {productname} repositories:
++
+
+----
+SUSEConnect -p sle-module-basesystem/15.1/x86_64
+SUSEConnect -p sle-module-server-applications/15.1/x86_64
+----
++
+To get all features working you should allow to install also recommended packages.
```

Comment:
Sometimes people create there own "golden SLES image". In this case the solver configuration allow already to install recommended packages.
I think we should say "verify that you allow installing recommended packages. Check the settings in ...."

Reply:
ok, submitted from the CL.